build.gradle

```diff
@@ -176,7 +176,7 @@ tasks.register("verifyVersions") {
  */
 
 boolean bwc_tests_enabled = false
-final String bwc_tests_disabled_issue = "https://github.com/elastic/elasticsearch/pull/65927" /* place a PR link here when committing bwc changes */
+final String bwc_tests_disabled_issue = "https://github.com/elastic/elasticsearch/pull/65952" /* place a PR link here when committing bwc changes */
```

Comment:
Could you remind me what happens after your change is integrated?
Will you revert this PR number to `65927`?

Reply:
I am tackling that offline with @jimczi, normally this field is empty, but we both have BWC at the moment.